src/server/api/user.ts

```diff
@@ -26,24 +26,24 @@ router.get('/me', async (req, res, next) => {
     return res.sendStatus(403);
   }
 
-  const integrationsFeed = await fetch(process.env.INTEGRATIONS_FEED_URL).then((res) =>
-    res.json() as Promise<Feed[]>
-  );
-
-  const userIntegrations = Object.keys(configuration).filter(key => key.endsWith('_INTEGRATION_ID') && !!configuration[key as keyof Config]).map(i => {
-    const envPrefix = i.replace('_INTEGRATION_ID', '')
+  try {
+    const integrationsFeed = await fetch(process.env.INTEGRATIONS_FEED_URL).then(
+      (res) => res.json() as Promise<Feed[]>
+    );
 
-    return {
-      feedId: (envPrefix || '').replace(/_/g, '-').toLowerCase(),
-      integrationId: configuration[i as keyof Config],
-    }
-  })
+    const userIntegrations = Object.keys(configuration)
+      .filter((key) => key.endsWith('_INTEGRATION_ID') && !!configuration[key as keyof Config])
+      .map((i) => {
+        const envPrefix = i.replace('_INTEGRATION_ID', '');
 
-  const list = await Promise.all(integrationsFeed.filter(entity => userIntegrations.find(i => i.feedId === entity.id)).map(async (entity) => {
-    const integrationId = userIntegrations.find(i => i.feedId === entity.id).integrationId
+        return {
+          feedId: (envPrefix || '').replace(/_/g, '-').toLowerCase(),
+          integrationId: configuration[i as keyof Config],
+        };
+      });
 
-    const response = await fetch(
-      `${fusebitIntegrationUrl}/${integrationId}/install?tag=fusebit.tenantId=${currentUserId}`,
+    const installsResponse = await fetch(
+      `${fusebitIntegrationUrl.split('/integration')[0]}/install?tag=fusebit.tenantId=${currentUserId}`,
```

Comment:
That's a _super_ awkward way of getting the root url.  I'd recommend composing it from scratch or helper rather than doing this.

Reply:
Hmm I think you're right. Currently we're getting ` FUSEBIT_INTEGRATION_URL` and setting it as a memory variable called `fusebitIntegrationUrl`, which is a problem if we want to target other api scopes. The cleanest solution for me is to get *only* the base url, which will look like this: `https://api.us-west-1.on.fusebit.io/v2/account/acc-3113066b371c48b4/subscription/sub-e0d98cb85ad64f24`, and then use it to target the different endpoints . This will require a small refactor here and a minimal change in the portal. We're working on it :)

Reply:
Changes made, what do you think now?